apps/prairielearn/assets/scripts/institutionAdminCourseInstanceClient.ts

```diff
@@ -0,0 +1,11 @@
+import { onDocumentReady } from '@prairielearn/browser-utils';
+
+onDocumentReady(() => {
+  document.querySelectorAll('.js-plan').forEach((plan) => {
+    const enabledCheckbox = plan.querySelector<HTMLInputElement>('.js-plan-enabled');
+    const enabledType = plan.querySelector<HTMLSelectElement>('.js-plan-type');
+    enabledCheckbox.addEventListener('change', () => {
+      enabledType.disabled = !enabledCheckbox.checked;
+    });
+  });
+});
```

Comment:
This is an exact copy of `institutionAdminGeneralClient.ts`. We previously talked about bringing over `behaviors/enable-by-checkbox.ts` from PT; maybe it's time to do that now?

Reply:
The same thing happened in PT where we started accumulating more and more of these until we added `enable-by-checkbox`. I don't have a firm opinion about when is the right time to bring it over.

Reply:
In the interest of shipping, I'll retain the duplication for now.